wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/ast2wdlom/AstToDeclarationElement.scala

```diff
@@ -2,20 +2,20 @@ package wdl.draft3.transforms.ast2wdlom
 
 import common.validation.ErrorOr.ErrorOr
 import wdl.draft3.parser.WdlParser
-import wdl.model.draft3.elements.{DeclarationElement, ExpressionElement, TypeElement}
+import wdl.model.draft3.elements.{SetDeclarationElement, ExpressionElement, TypeElement}
 import cats.syntax.apply._
 import cats.syntax.either._
 import wdl.draft3.transforms.ast2wdlom.EnhancedDraft3Ast._
 
-object AstToOutputDeclarationElement {
-  def convert(a: WdlParser.Ast): ErrorOr[DeclarationElement] = {
+object AstToDeclarationElement {
```

Comment:
ToL maybe this should be renamed as well for continuity? I'm sort of unclear on the reasoning for changing "Declaration" -> "SetDeclaration". I think InputDeclaration being its own separate entity makes sense because the expression parameter is optional, but a Declaration must by definition have an expression, so it's sort of implied that it's setting something in the parameters. "Set" seems superfluous.

Reply:
Yeah, I'd go along with that. I'll make the change back